content/applications/sales/sales/ebay_connector/setup.rst

```diff
@@ -1,170 +1,376 @@
+
+- Generate a marketplace account deletion/closure notification token. Navigate to
+  :menuselection:`Sales app --> Settings --> eBay` to start. Change the mode to
+  :guilabel:`Production`, and input random text values for the :guilabel:`Production App Key` and
+  for the :guilabel:`Production Cert Key`. Then click on :guilabel:`Generate Token` under the
+  :guilabel:`eBay Marketplace Account Deletion/Closure Notifications` section. This token will be
+  used during the setup on eBay for the deletion/closure notifications configuration.
+
+.. image:: setup/generate-token.png
+   :align: center
+   :alt: Generate a verification token in Odoo.
 
-.. image:: setup/setup02.png
-  :align: center
+Set up on eBay
+==============
 
-First choose if you want to use the production or the sandbox eBay Site. Then
-fill in the fields **Developer Key**, **Token**, **App Key**, **Cert Key**.
-Apply the changes.
+Set up eBay developer account
+-----------------------------
 
-Once the page is reloaded, you need to synchronize information from eBay. Push
-on **Sync countries and currencies**, then you can fill in all the other fields.
+To start, create an eBay developer account via `eBay's developer portal
+<https://go.developer.ebay.com/>`_. This site requires a different login and password than the eBay
+account, though the same email address can be used to register. The verification to create a
+developer account is around 24 hours.
 
-When all the fields are filled in, you can synchronize the categories and the
-policies by clicking on the adequate buttons.
+Set up eBay keyset
+------------------
 
-Accept account deletion notifications
-=====================================
+Once the eBay developer account is created, an application will need to be set up. To set up an
+application, start on `eBay's developer portal <https://go.developer.ebay.com/>`_ and navigate to
+the `Hi [username]` at top right of screen, then click on :guilabel:`Application Keysets`. A screen
+will appear with the prompt to :guilabel:`Enter Application Title`. This title can be no more than
+fifty characters. Enter the application name, then choose the appropriate environment (Sandbox or
+Production) to generate the first keyset. This application name will not be saved until the keyset
+is generated. Click on :guilabel:`Create a keyset` to generate the keyset.
 
-Since September 2021, **eBay requires supporting customer account deletion/closure notifications**.
-As such, when eBay receives an account request for deletion, all eBay partners must confirm the
-reception of the request and take further action if necessary.
+.. warning::
+   Once a Production Keyset is created, the keyset will be disabled. By default, the keyset will
+   been disabled, they can be activated by subscribing to the eBay Marketplace account deletion or
+   closure notifications or by applying to eBay for an exemption. On enabling, the database can make
+   5000 calls per day with this keyset.
```

Comment:
This is a rewrite for brevity and clarity based on my understanding of this paragraph. Feel free to accept only parts of the suggestion if it's not really correct (:
```suggestion
   The newly created *production keyset* is disabled by default. Activate it by subscribing to the
   eBay Marketplace 'account deletion or closure notifications' or by applying to eBay for an
   exemption. Upon enabling, the database can make 5000 calls per day using this keyset.
```